src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,130 @@
+package mate.academy.springboot.swagger.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+import mate.academy.springboot.swagger.dto.ProductRequestDto;
+import mate.academy.springboot.swagger.dto.ProductResponseDto;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.service.ProductService;
+import mate.academy.springboot.swagger.service.mapper.RequestDtoMapper;
+import mate.academy.springboot.swagger.service.mapper.ResponseDtoMapper;
+import mate.academy.springboot.swagger.util.SortUtil;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/products")
+public class ProductController {
+    private static final String[] TITLE = {"Bread", "Butter", "Cocoa",
+            "Coffee", "Apple", "Cucumber"};
+    private static final int MAX_PRICE = 10000;
+    private static final int NUMBER_OF_PRODUCTS = 50;
+    private static final Random RANDOM = new Random();
+    private final ProductService productService;
+    private final RequestDtoMapper<Product, ProductRequestDto> requestDtoMapper;
+    private final ResponseDtoMapper<Product, ProductResponseDto> responseDtoMapper;
+    private final SortUtil sortUtil;
+
+    public ProductController(ProductService productService,
+                             RequestDtoMapper<Product, ProductRequestDto> requestDtoMapper,
+                             ResponseDtoMapper<Product, ProductResponseDto> responseDtoMapper,
+                             SortUtil sortUtil) {
+        this.productService = productService;
+        this.requestDtoMapper = requestDtoMapper;
+        this.responseDtoMapper = responseDtoMapper;
+        this.sortUtil = sortUtil;
+    }
+
+    @PostConstruct
+    public void inject() {
+        for (int i = 0; i < NUMBER_OF_PRODUCTS; i++) {
+            productService.save(generateProduct());
+        }
+    }
+
+    @PostMapping("/create")
```

Comment:
/create is not needed here as you have the POST method which is responsible for the 'create' operation. same for others endpoints